packages/webapp/src/utils/utils.tsx

```diff
@@ -322,6 +322,9 @@ export const parseInput = (flow: Flow) => {
 };
 
 export function generateResponseModel(models: NangoSyncModel[], output: string, isSync: boolean): Record<string, any> {
```

Comment:
```suggestion
export function generateResponseModel(models: NangoSyncModel[], output: string | undefined, isSync: boolean): Record<string, any> {
```
The type is incorrect then, and that also means Flow's type is also wrong if `output`(returns) is optional

Reply:
Updated https://github.com/NangoHQ/nango/pull/1797/commits/71e47d002478a281ed8b820919d457903efdf529

output is optional already actually https://github.com/NangoHQ/nango/blob/master/packages/webapp/src/types.ts#L219

Reply:
EndpointReference.tsx is using `returns` not `output` (I don't know the difference between those 2 properties)

Reply:
`returns` is the v1 legacy property and `output` is the v2 property.